mongomock/filtering.py

```diff
@@ -16,7 +16,7 @@
     NoneType = type(None)
 
 try:
-    from bson import Regex
+    from bson import Regex, DBRef
```

Comment:
`mongomock` should work even if `pymongo` is not installed. Please fix what should happen in that case.